app/controllers/candidate_interface/new_references/email_address_controller.rb

```diff
@@ -0,0 +1,53 @@
+module CandidateInterface
+  module NewReferences
+    class EmailAddressController < BaseController
+      before_action :redirect_to_review_page_unless_reference_is_editable, :verify_email_is_editable
+      before_action :set_edit_backlink, only: %i[edit update]
+
+      def new
+        @reference_email_address_form = Reference::RefereeEmailAddressForm.build_from_reference(@reference)
+      end
+
+      def create
+        @reference_email_address_form = Reference::RefereeEmailAddressForm.new(referee_email_address_param)
+
+        if @reference_email_address_form.save(@reference)
+          redirect_to references_relationship_path(@reference.id)
+        else
+          track_validation_error(@reference_email_address_form)
+          render :new
+        end
+      end
+
+      def edit
+        @reference_email_address_form = Reference::RefereeEmailAddressForm.build_from_reference(@reference)
+      end
+
+      def update
+        @reference_email_address_form = Reference::RefereeEmailAddressForm.new(referee_email_address_param)
+
+        if @reference_email_address_form.save(@reference)
+          redirect_to return_to_path || references_review_unsubmitted_path(@reference.id)
+        else
+          track_validation_error(@reference_email_address_form)
+          render :edit
+        end
+      end
+
+    private
+
+      def referee_email_address_param
+        strip_whitespace params
```

Comment:
I think this was mentioned elsewhere but parenthesis would help readability here.